Week-3/assignment 3.4/stack.js

```diff
@@ -0,0 +1,16 @@
+const createStack = () => {
```

Comment:
The solution given is simple yet efficient. Also try implementing the solution using closures.